### ISSUE TITLE
Make AnkiDroid accessible with Talkback

### ISSUE THREAD

**Issue by eginhard** [ankidroid/Anki-Android#3795]
I just tried using AnkiDroid with Talkback and it's currently not usable at all. Many buttons and text fields are read out as "Button 68" or "Edit text" and the contents of the flashcard webview aren't accessible at all (although you can separately activate TTS for that in the advanced settings).

Any feedback on how to improve accessibility in general is greatly appreciated!

**Comment by ospalh** [comment]
> Any feedback on how to improve accessibility in general is greatly appreciated!

I once added a few texts for screen readers. Probably not enough.
The commit was  213d8af , what i did was adding `android:contentDescriptions` to the `.xml`files, e.g. [here](https://github.com/ankidroid/Anki-Android/commit/213d8afca1aeb6cbdef4e1244182a6927a023202?w=1#diff-09e0682f0600ba416572daa69786457aR35).

**Comment by gwildor28** [comment]
Hi timrae, I looked into the issue. I had multiple problems getting the layout in design view to render properly in Android Studio and thus I couldn't resolve the issue. I had problems with Android Support Design Widget Coordinator Layout, Navigation View Coordinator Layout and missing styles, etc. Are there any other issues within the context I outlined in my initial post that you think would be suitable for me? Ill try to find something in the meantime
